### PR TITLE
[styles] Add more icons

### PR DESCRIPTION
**Pull request by dvdmrtnz**
* Add icon to man_made=works. Closes https://github.com/organicmaps/organicmaps/issues/5874
* Add icon to craft=winery. Related to https://github.com/organicmaps/organicmaps/issues/2748
* Add icon to barrier=cycle_barrier. Closes https://github.com/organicmaps/organicmaps/issues/3922
* Add icon to amenity=vehicle_inspection. Closes https://github.com/organicmaps/organicmaps/issues/2621

| Day | Night |
|-|-|
|  |  |
|  |  |
|  |  |
|  |  |

### CODE REVIEW

**Comment by dvdmrtnz** [comment]
Simplified factory icon:

| Day | Night |
|-|-|
|  |  |

**Comment by pastk** [comment]
The new icons look great!
Except the cycle barrier maybe. Doesn't ring a bell for me (and I don't have better ideas either). I hope users will learn it after tapping on it.

Do you have an idea for a generic "craft=" icon? The current "stopping palm" icon is a terrible mismatch :)

**Review by pastk — COMMENTED** [review]
Looks good!

Please add `vehicle_inspection` to the vehicle style as well!

File: data/categories.txt [hunk 1]
```diff
@@ -5718,6 +5718,44 @@ sk:3poštový úrad
...
+it:Ispezione del veicolo
+ja:車検
+ko:차량검사
+mr:वाहन तपासणी
+nb:Inspeksjon av kjøretøy
+nl:Voertuiginspectie
+pl:Kontrola pojazdu
+pt:Inspeção veicular
+ro:Inspecția vehiculului
+ru:Осмотр транспортного средства
```
Comment (vng) [inline data/categories.txt:5749]: ru:Техосмотр автомобиля

File: data/categories.txt [hunk 2]
```diff
@@ -5718,6 +5718,44 @@ sk:3poštový úrad
...
+sk:Kontrola vozidla
+sv:Fordonsinspektion
+sw:Ukaguzi wa Gari
+th:การตรวจสอบยานพาหนะ
+tr:Araç muayenesi
+uk:Огляд автомобіля
```
Comment (vng) [inline data/categories.txt:5755]: Техогляд автомобіля

**Review by vng — COMMENTED** [review]
@pastk To discuss: Is it ok that barrier=lift_gate/block/bollard/... are drawn from z16, when many POIs only from z17 or even z18? Move barriers on z17?

**Comment by dvdmrtnz** [comment]
> Looks good!
> 
> Please add `vehicle_inspection` to the vehicle style as well!

Done

**Comment by pastk** [comment]
> Please add `vehicle_inspection` to the vehicle style as well!

... and it needs to be added to the vehicle prio.txt file too.

**Comment by pastk** [comment]
> @pastk To discuss: Is it ok that barrier=lift_gate/block/bollard/... are drawn from z16, when many POIs only from z17 or even z18? Move barriers on z17?

I think its ok, as barriers are important to see when navigating around. Also they're in the middle of roads usually and have small icons, so don't compete that much with POIs around.

**Comment by dvdmrtnz** [comment]
> > Please add `vehicle_inspection` to the vehicle style as well!
> 
> ... and it needs to be added to the vehicle prio.txt file too.

Done

**Review by pastk — APPROVED** [review]
Note: needs maps regen for the vehicle_inspection.

**Review by vng — APPROVED** [review]

**Pull request merged**